docs/skopeo.1.md

```diff
@@ -18,6 +18,10 @@ Most commands refer to container images, using a _transport_`:`_details_ format.
   default read from `$HOME/.kube/config`, which is set e.g. using
   `(oc login)`.
```

Comment:
Yeah, the implementation is actually in `containers/image/openshift` :) Accepting both is possible but can lead to surprising results e.g. in the policy (an unexpected section applies, with deduplication a new one can’t be added).

Ultimately the `X-Registry-Supports-Signatures` extension makes the `atomic:` transport obsolescent anyway.